ortho_seq_code/logo_plot.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import pandas as pd
+import logomaker as lm
+import matplotlib.pyplot as plt
+import ortho_seq_code.utils as utils
```

Comment:
Not sure how nit picky you want to be for imports but generally you do system packages, single blank line, 3rd party packages, single blank line, project level packages.